instaloader/__main__.py

```diff
@@ -250,7 +276,8 @@ def main():
     parser = ArgumentParser(description=__doc__, add_help=False, usage=usage_string(),
                             epilog="The complete documentation can be found at "
                                    "https://instaloader.github.io/.",
-                            fromfile_prefix_chars='+')
+                            fromfile_prefix_chars='+'
+                            )
```

Comment:
```suggestion
                            fromfile_prefix_chars='+')
```

(revert this unnecessary change)